src/test/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGeneratorTest.java

```diff
@@ -78,73 +77,90 @@ protected final String getNonCompilablePath(String filename) throws IOException
 
     @Test
     public void testIsProperUtilsClass() throws ReflectiveOperationException {
-        assertTrue("Constructor is not private", TestUtil.isUtilsClassHasPrivateConstructor(
-            JavadocPropertiesGenerator.class, false));
+        assertTrue(TestUtil.isUtilsClassHasPrivateConstructor(
+            JavadocPropertiesGenerator.class, false), "Constructor is not private");
     }
 
     @Test
-    public void testNonExistentArgument() throws Exception {
+    public void testNonExistentArgument(@SysErr Capturable systemErr, @SysOut Capturable systemOut)
+            throws Exception {
+        systemErr.capture();
+        systemOut.capture();
         JavadocPropertiesGenerator.main("--nonexistent-argument");
 
         final String expected = String.format(Locale.ROOT, "Missing required options "
                 + "[--destfile=<outputFile>, params[0]=<inputFile>]%n")
                 + USAGE;
-        assertEquals("Unexpected error log", expected, systemErr.getLog());
-        assertEquals("Unexpected output log", "", systemOut.getLog());
+        assertEquals(expected, systemErr.getCapturedData(), "Unexpected error log");
+        assertEquals("", systemOut.getCapturedData(), "Unexpected output log");
```

Comment:
`assertEquals("", systemOut.getCapturedData()`

May be change to `assertTrue(systemOut.getCapturedData().isEmpty())` here and below?

Reply:
Or even better
```
assertWithMessage("Unexpected output log")
    .that(systemOut.getCapturedData())
    .isEmpty();
```
But not in this PR. For this issue I would like the changeset to be as small as possible.